pkg/controllers/csi/provisioner/install.go

```diff
@@ -93,7 +94,9 @@ func (provisioner *OneAgentProvisioner) getDigest(dynakube dynatracev1beta1.Dyna
 		return "", err
 	}
 
-	return string(imageVersion.Digest), nil
+	digest, _ := strings.CutPrefix(string(imageVersion.Digest), "sha256:")
```

Comment:
Would it be good to handle the case if CutPrefix returns false? Or do you think it is not necessary?

Reply:
it;s not necessary., 
https://go.dev/play/p/71it3wq7o_W